source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/AdaptiveCardRenderer.java

```diff
@@ -67,10 +69,34 @@ public RenderedAdaptiveCard render(
         ICardActionHandler cardActionHandler,
         @Nullable IOverflowActionRenderer overflowActionRenderer,
         HostConfig hostConfig)
+    {
+        ChannelAdaptor channelAdaptor = new ChannelAdaptor.Builder().actionHandler(cardActionHandler).build();
+        return render(context, fragmentManager, adaptiveCard, channelAdaptor, overflowActionRenderer, null, hostConfig);
+    }
+
+    public RenderedAdaptiveCard render(Context context,
+                                       FragmentManager fragmentManager,
+                                       AdaptiveCard adaptiveCard,
+                                       @NonNull ChannelAdaptor channelAdaptor,
+                                       @Nullable IOverflowActionRenderer overflowActionRenderer,
+                                       @Nullable ActivityResultRegistry activityResultRegistry)
+    {
+        return render(context, fragmentManager, adaptiveCard, channelAdaptor, overflowActionRenderer, activityResultRegistry, defaultHostConfig);
```

Comment:
why is defaultHostConfig used here?

Reply:
Thanks for pointing out, somehow got it wrong during resolving conflicts. Updated and fixed.